include/modem/lte_lc.h

```diff
@@ -150,26 +150,35 @@ enum lte_lc_func_mode {
 
 enum lte_lc_evt_type {
 	/** @brief Event received carrying information about the modems network registration
-	 *         status.
-	 *         Payload is of type @ref lte_lc_nw_reg_status.
+	 *         status. The event type has no associated payload.
 	 */
 	LTE_LC_EVT_NW_REG_STATUS,
+
 	/** @brief Event received carrying information about PSM updates. Contains PSM parameters
 	 *	   given by the network.
-	 *         Payload is of type @ref lte_lc_psm_cfg.
+	 *         The associated payload is the psm_cfg member of type @ref lte_lc_nw_reg_status
+	 *	   in the event.
 	 */
 	LTE_LC_EVT_PSM_UPDATE,
+
 	/** @brief Event received carrying information about eDRX updates. Contains eDRX parameters
+	 *	   given by the network.out PSM updates. Contains PSM parameters
```

Comment:
This line seems to be copy-pasted here by mistake? :)

Reply:
Never a PR without these copy/paste mistakes..